controller/command/command.go

```diff
@@ -65,6 +66,10 @@ func (self *LocalDispatcher) IsLeaderOrLeaderless() bool {
 	return true
 }
 
+func (self *LocalDispatcher) IsLeaderless() bool {
+	return false
```

Comment:
Why is this always fase?

Reply:
LocalDispatcher is only used when running in non-HA mode, without raft, so there's only one node which is the equivalent of being the leader.